node/src/components/event_stream_server.rs

```diff
@@ -77,8 +87,10 @@ impl EventStreamServer {
         let (sse_data_sender, sse_data_receiver) = mpsc::unbounded_channel();
 
         // Event stream channels and filter.
+        let broadcast_channel_size = config.event_stream_buffer_length / 100
```

Comment:
Is that value in bytes? If so, can we add the suffix `_bytes` to it?

Reply:
No, they both measure event counts.